src/CommandingOfficers/Qis.java

```diff
@@ -0,0 +1,234 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+import Engine.GamePath;
+import Engine.GamePath.PathNode;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.MapChangeEvent;
+import Engine.UnitMods.UnitFightStatModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitMovementModifier;
+import Terrain.Environment;
+import Terrain.Environment.Weathers;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Units.Unit;
+import Units.UnitContext;
+import Units.MoveTypes.MoveType;
+
+public class Qis extends Commander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Qis");
+      infoPages.add(new InfoPage(
+          "Commander Qis (pronounced 'keese') has spent his adult life fleeing cold weather, and failing to escape it.\n" +
+          "Whether the cause is simply incredible misfortune or some sort of curse is unclear... he is quite curt on the subject.\n" +
+          "What is known is that Qis has become incredibly adept at using the ever-present snow to his advantage."));
+      infoPages.add(new InfoPage(
+          "Passive:\r\n" +
+          "Watery terrain slows your non-naval units, but all units have perfect movement in snow.\n" +
+          "The path each unit takes on your turn is coated with snow until your next turn."));
+      infoPages.add(new InfoPage(
+          GroovinMovin.NAME+" ("+GroovinMovin.COST+"):\n" +
+          "+"+GroovinMovin.BUFF+" move for all units\n" +
+          "Snow trails left this turn last +1 turn for each space moved so far"));
+      infoPages.add(new InfoPage(
+          TrailOfChill.NAME+" ("+TrailOfChill.COST+"):\n" +
+          "+"+TrailOfChill.BUFF+" move for all units\n" +
+          "All snow on the map will last "+TrailOfChill.DURATION+" turns"));
```

Comment:
On larger maps, this seems like it would just turn into perpetual snow.

Reply:
It could. The neat bit is that he can only make snow where his dudes go, though - it just interacts in fun ways with other COs.
The SCOP is intentionally a bit more niche than COP.